tests/integration/timeout_test.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import pytest
+
+import libnmstate
+from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceType
+from libnmstate.schema import InterfaceState
+from libnmstate.schema import LinuxBridge
+from libnmstate.schema import VLAN
+
+
+@pytest.mark.skip(reason="This test would take a lot of time.")
```

Comment:
Please use the similar way like https://github.com/nmstate/nmstate/pull/776 did for pytest slow marker.

Reply:
Please also use https://docs.travis-ci.com/user/customizing-the-build#Fast-Finishing which allow  travis to mark test as passed as soon as mandatory test passed.

Reply:
Done, thanks!